pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -184,29 +184,35 @@ func (p *staticPolicy) Name() string {
 }
 
 func (p *staticPolicy) Start(s state.State) error {
```

Comment:
suggested commit message:
```
move default cpuset initialization from state validation

While it works, intializing the default cpuset during the state validation at policy start time
is unnecessarily convoluted and couples code paths.
Initialization should happen separately from the state validation, and ideally the state validation
should only consume the state.State object in read-only mode, without modifying it.
```

Reply:
> Initialization should happen separately from the state validation, and ideally the state validation
> should only consume the state.State object in read-only mode, without modifying it.

That's exactly how it works at the moment (after incorporating this suggestion you made earlier).

Reply:
perfect, please amend the commit message of this commit to record the information above. Feel free to rephrase as you see fit.

Reply:
Thanks for the update. Unfortunately the commit message looks garbled.
it should be like
```
summary line

longer description over multiple lines
```
please note the empty line after the summary, and please use active form in the summary "move code..."
No need to append "suggested by fromani".